docs/openapi.json

```diff
@@ -1004,6 +1010,145 @@
         },
         "summary": "Ping the server (healthcheck)"
       }
+    },
+    "/api/users": {
+      "get": {
+        "description": "Get user information.",
+        "operationId": "get_user",
+        "parameters": [
+          {
+            "description": "Required. The email of the user.",
+            "in": "query",
+            "name": "email",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "description": "Not required. UUID of the user.",
+            "in": "query",
+            "name": "id_",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "description": "Required. API key of the admin.",
+            "in": "query",
+            "name": "api_key",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "User was found. Returns all user info stored.",
```

Comment:
user info stored->user's information? at some point it will be shown in "extense" API docs so no need to save characters.